addons/website/static/src/js/menu/mobile_view.js

```diff
@@ -50,7 +50,10 @@ var MobileMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
      * @private
      */
     _onMobilePreviewClick: function () {
-        new MobilePreviewDialog(this, {
+        if (this.mobilePreview && !this.mobilePreview.__closed) {
```

Comment:
Is that possible? If possible in the dblclick case, maybe we should simply use the famous 'async' keyword on top of the `_onMobilePreviewClick` string in the `events` object definition.

Reply:
It's now possible because of the accesskey, you can just keep "clicking" the button even though it's behind the backdrop using the shortcut.

Reply:
Meh. Is not that a bug that should be fixed generically? So in the backend, if I open a modal, I am still able to click on the EDIT/SAVE buttons which are behind?

(note: in any case, you should not use `__closed` but `isDestroyed()` instead (which are the same for odoo modals)).

Reply:
> So in the backend, if I open a modal, I am still able to click on the EDIT/SAVE buttons which are behind?

Yup. Not sure why you'd consider it a bug though. In most cases, it does exactly what's expected.

Reply:
"In most cases" is enough to consider it as a bug but maybe that's just me... :fearful: In fact, if we want to support that, we can make this a toggle then. Instead of doing nothing if the modal is opened, close it? Maybe check with @sbuhl

Reply:
@qsm-odoo Alright, discussed with sbu, he said toggle for mobile preview is better than ignoring subsequent click, so I changed that, changed to use `isDestroyed()` instead of `__closed`, and additionally, prevented the publish button from being used from edit mode through the shortcut as he asked me.

Since being able to use buttons that you're not normally able to click is a more general problem, sbu said he'd report the issue to the framework team and they'll decide what they want to do about it.